requirements-dev.txt

```diff
@@ -61,3 +61,4 @@ pyamg>=5.0.0
 git+https://github.com/facebookresearch/segment-anything.git@6fdee8f2727f4506cfbbe553e23b895e27956588
 onnx_graphsurgeon
 polygraphy
+e3nn
```

Comment:
Hi, thank you for the feedback. Could you elaborate more on what core I should integrate the feature into in the monai folder? Any guidance would be much appreciated.

Reply:
Hi @djoca77, you may need to minimize code repetition as much as possible by organizing related components into appropriate structures. For instance, place convolution operations within the network block, and replace loading .nii files with`LoadImage` from MONAI. If you find no highly generic components to extract, consider adding this as an example in the tutorial for clarity and reference. Thanks.

Reply:
Hi @KumoLiu thank you for the help. I think for now I'd like to add this as an example of the equivariant feature extraction instead of a fully fledged functionality. I can make some changes to use some of the MONAI generic functions. Could you give me some pointers on contributing this as an example. I wonder whether I should just add this to the tutorial as you suggested. Thanks.